Updated activity details and some cosmetic changes (#1)
supernord/website: pages/activities/infrastructure_roadmap.md

---
title: Infrastructure Roadmap
type: activity_inprogress
---

### Details:

> The infrastructure roadmap document intends to establish computing infrastructure priotities for the national structural biology community.

Reviewer: ```suggestion
The infrastructure roadmap document _"describes the existing national landscape, identifies and prioritizes critical research bottlenecks, and proposes a national strategy to unlock the immense potential of computational structural biology for Australian researchers"_ (text copied from roadmap draft).
```

### Completed:

> - [x] Initial draft of infrastructure roadmap.

Reviewer: ```suggestion
- [x] Initial draft of infrastructure roadmap
- [x] Initial community feedback incorporated, content updated, implementation section added
- [x] Review and updates based on feedback from co-authors and academic panel
- [x] Review and updates based on feedback from community and BioCommons infrastructure partners
- [x] Review and updates based on feedback from international experts
```

- [x] Review from community academic panel.

> - [x] Review from community members and infrastructure partners.

Reviewer: ```suggestion
```

> - [x] International review.

Reviewer: ```suggestion
```
